### PR TITLE
Fix of ABI Problem, commodity forward notional and userdoc improvements

### PR DESCRIPTION
**Pull request by rkapl123**
- fixed userdoc simulation.xml example AggregationScenarioDataCurrencies and AggregationScenarioDataIndices.

### CODE REVIEW

**Comment by kohnech** [comment]
Wasn't the ABI issue discussed in https://github.com/OpenSourceRisk/Engine/issues/160
?

File: OREAnalytics/test/CMakeLists.txt [hunk 1]
```diff
@@ -32,6 +32,9 @@ target_link_libraries(orea-test-suite ${ORED_LIB_NAME})
 target_link_libraries(orea-test-suite ${OREA_LIB_NAME})
 target_link_libraries(orea-test-suite ${Boost_LIBRARIES} ${RT_LIBRARY})
 
+# avoid ABI mismatch between precompiled boost.log and this compiler run...
```
Comment (kohnech) [inline OREAnalytics/test/CMakeLists.txt:35]: Maybe add comment that this is fix for boost >1.73 both manual and pre-builds.

Comment (rkapl123) [reply OREAnalytics/test/CMakeLists.txt]: I've added a hint for 1_73 now.

**Review by kohnech — APPROVED** [review]
👍

**Comment by rkapl123** [comment]
Actually the change to [OREAnalytics/test/CMakeLists.txt](https://github.com/OpenSourceRisk/Engine/pull/204/files/3524a6dc58948516ed235202b459662ba14f1bf3#diff-6375d9113bf6185c4ac33c3c7919ae47035de65a7b8fee454eb0d2de27a012c9) is obsolet with the addition of https://github.com/OpenSourceRisk/Engine/commit/e2f19e836795f13cd47f655c6b368db449e1984d

**Pull request merged**